### DISCUSSION TITLE
Fdroid and permissions

### DISCUSSION THREAD

**Discussion by Wayneaswell** [KillerInk/FreeDcam#174] [Q&A]
I noticed that the Fdroid download has a lot of potentially invasive permissions, and the package is named Fdroid.  Is Fdroid adding it's own encapsulation with extra permissions?

**Comment by KillerInk** [comment]
fdroid take the source as it is and sign the apk with its own key. so no they dont add extra permissons
bevor L oems used permissions to lock their extended camera features.

that are the permissions that can get used by freedcam.

https://github.com/KillerInk/FreeDcam/blob/39e23436ae8a929fc7933c745bfac1eda3cf2680/app/src/main/AndroidManifest.xml#L6-L39

changing wifi and view availible wifis is needed for sony remote api and also full network access.
fine and corse location are for gps.

**Comment by Wayneaswell** [comment]
Thank you for this.  I constantly get an issue of whether t voiding and
loosing settings from handling it. I've asked open camera in the past
to.implement a plug ins for all features, including codecs, and and
configurable controls and WYSIWYG on screen interface, but they were not
interested.  But, people can easily do pro video plugins and interface like
that.  I have thought of hiring African programmers to make open camera
into a plug in routing shell, or do a new app.   If Apple introduced ProRes
on their phone today. It will be time for an droid to do this too.


Thank you.


Wayne.
